docs/signature/README.md

```diff
@@ -46,48 +46,70 @@ openssl req \
+```
+
 ### Signature Persisted within an OCI Artifact Enabled Registry
 
-Both values are persisted in a `signature.json` file. The file would be submitted to a registry as an Artifact with null layers.
-The `signature.json` would be persisted within the `manifest.config` object
+All values are persisted in a `signature.jwt` file. The file would be submitted to a registry as an Artifact with null layers.
+The `signature.jwt` would be persisted within the `manifest.config` object
 
 ``` SHELL
 oras push \
   registry.example.com/hello-world:v1.0 \
-  --manifest-config signature.json:application/vnd.cncf.notary.config.v2+json
+  --manifest-config signature.json:application/vnd.cncf.notary.config.v2+jwt
```

Comment:
:thumbsup: perfect! I was looking if you caught that we would change the artifact `config.mediaType` extension.